src/main/kotlin/no/nav/familie/ba/soknad/api/controllers/SøknadNewWip.kt

```diff
@@ -23,3 +27,42 @@ data class SøknadNewWip(
     val teksterUtenomSpørsmål: Map<SpørsmålId, Map<Locale, String>>,
     val originalSpråk: Locale
 )
+
+data class Søker(
+    val ident: Søknadsfelt<String>,
+    val navn: Søknadsfelt<String>,
+    val statsborgerskap: Søknadsfelt<List<String>>,
+    val adresse: Søknadsfelt<SøknadAdresse>,
+    val sivilstand: Søknadsfelt<SIVILSTANDTYPE>,
+    val spørsmål: Map<String, Søknadsfelt<Any>>,
+    val nåværendeSamboer: Søknadsfelt<NåværendeSamboer>?,
+    val tidligereSamboere: List<Søknadsfelt<TidligereSamboer>>,
+    val utenlandsperioder: List<Søknadsfelt<Utenlandsopphold>> = listOf(),
+    val andreUtbetalingsperioder: List<Søknadsfelt<Utbetalingsperiode>> = listOf(),
+    val arbeidsperioderUtland: List<Søknadsfelt<Arbeidsperiode>> = listOf(),
+    val arbeidsperioderNorge: List<Søknadsfelt<Arbeidsperiode>> = listOf(),
+    val pensjonsperioderNorge: List<Søknadsfelt<Pensjonsperiode>> = listOf(),
+    val pensjonsperioderUtland: List<Søknadsfelt<Pensjonsperiode>> = listOf()
+)
+
+data class Arbeidsperiode(
+    val arbeidsperiodeAvsluttet: Søknadsfelt<String?>,
```

Comment:
Skal nok være sånn men bare lurer på hvorfor alle disse er av typen `String?`

Reply:
Vi ser at  spørsmål er av typen `val spørsmål: Map<String, Søknadsfelt<Any>>,`, kunne vurdert å ha en egen type som var JA NEI VET_IKKE, men vi har det ikke for spørsmålene slik de er nå